locales/en.json

```diff
@@ -219,7 +219,7 @@
     "user_info_failed" : "Unable to retrieve user information",
     "user_home_creation_failed" : "Unable to create user home folder",
 
-    "package_unexpected_error" : "An unexpected error occured with the package '{pkgname}'",
+    "package_unexpected_error" : "An unexpected error occured in the package '{pkgname}'",
```

Comment:
This message is printed when *pkgname* - which is generally a Debian package - cannot be installed / upgraded / removed and unexpectedly. Is *in the package* mean that? I really don't want to question your English but just learn it! :)